mavros_msgs/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(msg_files
   msg/RTKBaseline.msg
   msg/RadioStatus.msg
   msg/State.msg
+  msg/StatusEvent.msg
```

Comment:
That list should be generated by cog.

Reply:
Is there any readme on how to do so? I installed cog, but running the `mavros_cog.py` script does not do anything...

Reply:
ok thanks, should be good now :ok_hand: